mockserver-core/src/main/java/org/mockserver/configuration/ConfigurationProperties.java

```diff
@@ -256,30 +256,24 @@ public static void overrideLogLevel(String level) {
 
     private static void overrideLogLevelWithReflection(String level, String loggerName) {
         Logger logger = LoggerFactory.getLogger(loggerName);
-
+        Class loggerClass = logger.getClass();
         try {
-            // check if logback-classic is being used
-            Class logbackLevelClass = ConfigurationProperties.class.getClassLoader().loadClass("ch.qos.logback.classic.Level");
-
-            // convert string to log level
-            Method toLevelMethod = logbackLevelClass.getDeclaredMethod("toLevel", String.class);
-            toLevelMethod.setAccessible(true);
-            Object levelInstance = toLevelMethod.invoke(logbackLevelClass, level);
-
-            // update log level
-            Method setLevelMethod = logger.getClass().getDeclaredMethod("setLevel", logbackLevelClass);
-            if (setLevelMethod != null) {
-                setLevelMethod.invoke(logger, levelInstance);
-            }
-        } catch (Exception e) {
-            ConfigurationProperties.logger.warn("Exception updating logging level using reflection, likely cause is Logback is not on the classpath");
-        }
+            // check if SimpleLogger is used (i.e. in maven plugin)
+            if (!loggerClass.getName().equals("org.slf4j.impl.SimpleLogger")) {
```

Comment:
Basically I just refactored this to handle whether we're using simple logger or not and only pass a warning to the client if there's a break.

We have a 0 warnings policy on our team and I can't seem to find any other way of suppressing the warnings other than this refactor.  Would be good, long-term, to also refactor to add log4j support, but we don't use log4j